pyspecProcScripts/phasing.py

```diff
@@ -199,7 +199,7 @@ def hermitian_function_test(
     s,
     direct="t2",
     frq_range=(-2.5e3, 2.5e3),  # assumes signal is somewhat on resonance
-    aliasing_slop=3,
+    aliasing_slop=2, # if this is set to 3 rather than 2, the IR version is failing.  That shouldn't be happening!  (Equivalently, if we set ini_delay to 3ms in 7093, this also fails!)
```

Comment:
I'm sorry but there are two things going on here at the same time:

- I establish a new convention, where the selection range, which is a bit hard-coded is removed in favor of an "aliasing slop" -- defined in my new docstring
- If I use either this new convention or the old convention, setting the initial part of the slice even slighly far out causes everything to fail -- I don't yet understand why this is, and hope you might have some ideas.

Reply:
okay, my thought is that it was working even if it was taking extra kwargs. We should merge the updated version before you added this aliasing_slop since this is adding more issues and then we can make a separate PR for incorporating it. This PR I feel is getting really convoluted since it was working prior to these aliasing_slop changes...